sql/moz-fx-data-shared-prod/telemetry/desktop_eng_agg/view.sql

```diff
@@ -0,0 +1,7 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.telemetry.desktop_eng_agg`
```

Comment:
Let's not abbreviate engagement

Reply:
Sounds good, I renamed the 2 views and 2 tables with engagement fully spelled out